### DISCUSSION TITLE
Add `Not Completed` button

### DISCUSSION THREAD

**Discussion by V1C70RYG0D** [solodit/feedback#17] [New Feature Requests]
please add a  button so that i can see what are findings i haven't completed yet, like if i start reading from the start then i have to go to page 15 or 50 that takes lot of time and will increase in future , so i want a feature which can give me my next not completed finding starting from page1 :) 
Thanks

**Comment by wang1252** [comment]
@V1C70RYG0D 

We will add the `Note Completed` filter.
Thank you for your feedback.

**Comment by hans-cyfrin** [comment]
We now have a way to filter "unread" findings. Thanks for the suggestion.
